usaon_vta_survey/routes/user.py

```diff
@@ -13,18 +13,25 @@ def load_user(user_id: str) -> User:
     return User.query.get(user_id)
 
 
-@app.route('/profile/<user_id>', methods=['POST', 'GET'])
-def profile(user_id: str):
+def _validate_role_change(user: User, form) -> None:
+    if not form.data['role_id'] == user.role_id:
+        if not current_user.role_id == 'admin':
+            raise RuntimeError("Only admins can edit users roles.")
```

Comment:
:pinched_fingers: 

Instead of a nested `if`, could also consider `if not X and not Y` or `if not (X or Y)`. Whatever is most readable!